WcaOnRails/app/views/layouts/application.html.erb

```diff
@@ -59,7 +59,7 @@
   <%= csrf_meta_tags %>
   <%= csp_meta_tag %>
   <%= auto_discovery_link_tag(:rss, rss_url(format: :xml)) %>
-  <% if ENVied.WCA_LIVE_SITE %>
+  <% if EnvVars.WCA_LIVE_SITE? %>
```

Comment:
Why the question mark here but not anywhere else?

Reply:
Same as DISABLE_BULLET